src/Print.hs

```diff
@@ -9,6 +10,9 @@ import Data.List
 import Syntax
 import Syntax.TimeAgnostic
 
+import qualified Syntax.Annotated as Annotated
+import qualified Syntax.TimeAgnostic as Agnostic
```

Comment:
We don't need to mention `Annotated` here, we can just say `Exp Timed` instead. This module already knows the internal representation.

```suggestion
```